app/controllers/short_url_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class ShortUrlController < ApplicationController
+  def accept_assignment
```

Comment:
Can we update these actions to `assignment_invitation` and `group_assignment_invitation`

This action isn't actually accepting anything so it's kind of misleading.